python-sdk/src/astro/sql/operators/raw_sql.py

```diff
@@ -44,9 +59,20 @@ def execute(self, context: Context) -> Any:
                 "backend."
             )
 
-        if self.handler and self.database_impl.sql_type == "delta":
+        # ToDo: Currently, the handler param in run_sql() method is only used in databricks all other databases are
+        # not using it. Which leads to different response types since handler is processed within `run_sql()` for
+        # databricks and not for other databases. Also the signature of `run_sql()` in databricks deviates from base.
+        # We need to standardise and when we do, we can remove below check as well.
```

Comment:
Can you create a Github issue for it too so that @phanikumv and you can prioritize it in one of the future sprints

Reply:
Sure, will add this.

Reply:
Added ticket to keep track -  https://github.com/astronomer/astro-sdk/issues/1664